package_modules/apt_get

```diff
@@ -0,0 +1,259 @@
+#!/usr/bin/python
+
+import sys
+import os
+import subprocess
+import re
+
+dpkg_cmd = "/usr/bin/dpkg"
+dpkg_deb_cmd = "/usr/bin/dpkg-deb"
+dpkg_query_cmd = "/usr/bin/dpkg-query"
+dpkg_output_format = "Name=${Package}\nVersion=${Version}\nArchitecture=${Architecture}\n"
+dpkg_status_format = "Status=${Status}\n" + dpkg_output_format
+
+apt_get_options = ["-o", "Dpkg::Options::=--force-confold", "-o", "Dpkg::Options::=--force-confdef", "-y", "--force-yes"]
+apt_get_cmd = "/usr/bin/apt-get"
+
+os.environ['DEBIAN_FRONTEND'] = "noninteractive"
+os.environ['LC_ALL'] = "C"
+
+if (os.environ.get('CFENGINE_TEST_DPKG_CMD')):
+    dpkg_cmd = os.environ['CFENGINE_TEST_DPKG_CMD']
```

Comment:
It would be cleaner and simpler to write
```
dpkg_cmd = os.environ.get('CFENGINE_TEST_DPKG_CMD', "/usr/bin/dpkg")
```
instead of initially setting to `"/usr/bin/dpkg"` and then conditionally calling `os.environ.get()` to decide whether to then over-write the initial setting with the `os.environ[]` value.
The .get(key, default) of a dictionary supplies the given default as value if the key is absent.

Also, in python, if doesn't need parentheses around its condition !
(... but the above simplification shall not only speed up this code but also get rid of the need for the conditional.)